src/sentry/integrations/bitbucket_server/webhook.py

```diff
@@ -91,12 +92,12 @@ def __call__(
                     project_name, repo_name, from_hash, change.get("toHash")
                 )
             except ApiHostError:
-                return HttpResponse(status=409)
+                raise SuspiciousOperation()
             except ApiUnauthorized:
-                return HttpResponse(status=400)
+                raise SuspiciousOperation()
             except Exception as e:
                 sentry_sdk.capture_exception(e)
-                return HttpResponse(status=400)
+                raise SuspiciousOperation()
```

Comment:
This may be more of a 500 case. Can we definitively give a response when this happens?

Reply:
like by reraising the exception?

Reply:
Yeah that should be fine. By default these will create sentry issues when you reraise right?

Reply:
yes it should